packages/mds-api-helpers/index.ts

```diff
@@ -17,13 +17,14 @@
 import urls from 'url'
 import express from 'express'
 import { isInsideBoundingBox } from 'mds-utils'
-import { VehicleEvent, Device, Telemetry, BoundingBox, EVENT_STATUS_MAP, VEHICLE_STATUSES } from 'mds-types'
+import { VehicleEvent, Device, Telemetry, BoundingBox, EVENT_STATUS_MAP, VEHICLE_STATUSES, VEHICLE_STATUS, VEHICLE_EVENT } from 'mds-types'
 import log from 'mds-logger'
 import db from 'mds-db'
 import cache from 'mds-cache'
 import { CacheReadDeviceResult } from 'mds-cache/types'
+import { StringifiedEventWithTelemetry } from 'packages/mds-cache/types';
 
-export async function getVehicles(
+export async function getVehiclesDb(
```

Comment:
At this point, I would be inclined to just move this code back to `agency` and `audit` respectively.

Reply:
Agreed.